### PR TITLE
Bump librocksdb-sys up to 6.11.4

### PR DESCRIPTION
**Pull request by aleksuss**

### CODE REVIEW

**Review by stanislav-tkach — APPROVED** [review]

**Comment by stanislav-tkach** [comment]
As far as I can see, there is `6.13.2` version available.

**Comment by aleksuss** [comment]
Yeah. Obviously, it makes sense to update up to `6.13.2`.

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 ## [Unreleased]
 
 * Add `DB::cancel_all_background_work` method (stanislav-tkach)
+* Bump `librocksdb-sys` up to 6.13.3 (aleksuss)
```
Comment (ordian) [inline CHANGELOG.md:6]: I'm confused a bit, this bump to 6.11.4 was done in #452 (and is published)
what this PR did is fixed clippy warnings

Comment (aleksuss) [reply CHANGELOG.md:6]: Yeah. You're right. There was some troubles with compilation rocksdb 6.13.3 and therefore I accidentally decreased a version to 6.11.4. Thanks for the notify. I'll bump librocksdb-sys version in near future.

Comment (ordian) [reply CHANGELOG.md:6]: Thanks for the clarification, so you've left it out for "future work" :)

Comment (aleksuss) [reply CHANGELOG.md:6]: Exactly :)

**Pull request merged**